ui/app/AppLayouts/Wallet/panels/WalletHeader.qml

```diff
@@ -36,12 +36,10 @@ Item {
 
     property bool dAppsEnabled: true
     property bool dAppsVisible: true
-    property bool walletConnectEnabled: true
-    property bool browserConnectEnabled: true
```

Comment:
No longer needed?

Reply:
no longer needed here. The reason is that I've moved the connect popups living in `DappsComboBox.qml` (the dapps button found in the header) in the DAppsWorkflow where the other dapps popups are implemented. These flags were needed for the connect popups.